src/app/ipv/middleware.js

```diff
@@ -44,13 +44,17 @@ async function journeyApi(action, req) {
     action = action.substr(1);
   }
 
+  if (action.startsWith("journey/")) {
+    action = action.substr(8);
+  }
```

Comment:
We should really be able to get rid of this, but I think we'll still get a `/journey/` prefix on journey events coming from the backend. (In future we should be able to eliminate these entirely - it's only really error scenarios that do this...)